src/parsing-2.html

```diff
@@ -1,66 +1,24 @@
-   
+
    <section>
       <h3 id="parsing_usingdtd">Using the MathML DTD</h3>
```

Comment:
I would remove the paragraph. Maybe it would be good enough to link to a program that can be used to generate a DTD from RelaxNG

Reply:
I cut it down a lot from how it as in mathml3, I'll cut it down a bit more in the following checkin.

Reply:
@Andreg-p and I were trying to use DTD based MathML validation in one project, but it was a dead-end road. We restarted with stylesheet-based validation (https://github.com/ag-gipp/MathMLTools), which worked, but was too slow for use on large datasets. Eventually, I gave up the idea of validating everything and tried to deal with less structured material.

I think it would be better to have one good way to fully validate MathML, rather than several approaches that do only some validation.